.travis.yml

```diff
@@ -32,7 +32,8 @@ env:
   - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB"  # ODOO_REPO usage example
   - VERSION="6.1" INCLUDE="test_module,second_module"
   - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="17" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
-  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="18" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
+  - VERSION=master LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="20" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
```

Comment:
You are right!
In this case we need to test a version that not is number.
And now we have splitted lint test and server test.
Then IMHO the problem in the past with master server test not is a problem now with master lint server.